lib/galaxy/model/store/__init__.py

```diff
@@ -37,14 +38,32 @@
 GALAXY_EXPORT_VERSION = "2"
 
 
+class ImportDiscardedDataType(Enum):
+    # Don't allow discarded 'okay' datasets on import, datasets will be marked deleted.
+    FORBID = 'forbid'
+    # Allow datasets to be imported as experimental DISCARDED datasets that are not deleted if file data unavailable.
+    ALLOW = 'allow'
+    # Import all datasets as discarded regardless of whether file data is available in the store.
+    FORCE = 'force'
+
+
+DEFAULT_DISCARDED_DATA_TYPE = ImportDiscardedDataType.FORBID
+
+
 class ImportOptions:
+    allow_edit: bool
+    allow_library_creation: bool
+    allow_dataset_object_edit: bool
+    discarded_data: ImportDiscardedDataType
```

Comment:
Wouldn't you type the __init__ signature normally ? Or should this be a dataclass ?